### ISSUE TITLE
Possibility to fit a parametric form to the gamma's in Henery model 

### ISSUE THREAD

**Issue by BenoitLondon** [shabbychef/ohenery#3]
For example, gamma's seem to decrease with the position, and I've read they could depend on the number of runners as well in an article form John Bacon-Shone (I can't find the reference)
cf:
LO(i, j|k) = λ(k,n) log(pi/pj), with λ(k,n) ≈ μ^k in Bacon-Shone article
where lambda is gamma in your formulation of Henery's model

Not sure how doable it is as it can't be too complex I guess?

but the dependence on number of runners makes sense to me

Thank you for that cool package! I might open more issues... :)

**Comment by shabbychef** [comment]
Thanks for opening the issue. I do not think this would be too hard to implement. I have two requests, however:
1. find me a reference to the article.
2. suggest a name for the code. (I am impartial to `bashsm` for "BAcon-SHone Soft Max".)

**Comment by BenoitLondon** [comment]
I saw this is in this chapter  

https://onlinelibrary.wiley.com/doi/10.1002/9780470400531.eorms0386

![image](https://github.com/shabbychef/ohenery/assets/7107193/68d72c0d-c109-4a1e-a18e-9a1c72d50ffe)

but the references are: 
`Lo, VSY; Bacon-Shone, J; Busche, K. (1992). A modified racetrack betting system. 18, 1–14. http://hdl.handle.net/10722/60980`

`Lo, V. S. Y., Bacon-Shone, J., & Busche, K. (1995). The Application of Ranking Probability Models to Racetrack Betting. Management Science, 41(6), 1048–1059. https://doi.org/10.1287/mnsc.41.6.1048`

For the implementation, I am not sure we need a new name, it's just an extension of your Henery model as I see it.
maybe just an argument like `gamma_method = c("discrete", "LBS")`  discrete being the current implementation and LBS the power shape, not sure about how to handle the number of runners though...

**Comment by shabbychef** [comment]
The Lo _et al_ 1995 article talks about a "discount model", introduced in their 1992 paper. The discount model depends on some parameter `r` which would have to be fit. However I do not see the `mu^k` approximation there.

**Comment by BenoitLondon** [comment]
yeah me neither I could not find that exact formulation of `mu^k` in the references...
